projects/core/src/user/store/actions/user-replenishment-orders.action.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { ReplenishmentOrderList } from '../../../model/replenishment-order.model';
+import { StateUtils } from '../../../state/utils/index';
+import { USER_REPLENISHMENT_ORDERS } from '../user-state';
+import { UserActions } from './index';
+
+const mockUserReplenishmentOrder: {
+  userId: string;
+  pageSize: number;
+  currentPage: number;
+  sort: string;
+} = {
+  userId: 'test@sap.com',
+  pageSize: 5,
+  currentPage: 1,
+  sort: 'byDate',
+};
+
+const mockUserReplenishmentOrders: ReplenishmentOrderList = {
```

Comment:
since these constant are only used in one specific `describe` function, lets put snap them in there